### PR TITLE
Parsed request context to bpm

### PR DESCRIPTION
**Pull request by hsong-rh**
Parsed request's context to get/set requester identity.

### CODE REVIEW

File: project.repositories [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
   <repositories>
     <repository>
-      <include>true</include>
+      <include>false</include>
       <metadata>
-        <id>central</id>
-        <url>https://repo.maven.apache.org/maven2</url>
-        <source>PROJECT</source>
+        <id>local</id>
+        <url>/Users/hsong/.m2/repository</url>
```
Comment (bzwei) [inline project.repositories]: should this be checked in?

File: src/main/java/com/redhat/management/approval/EmailDispatcher.java [hunk 2]
```diff
@@ -10,112 +10,114 @@
...
+    }
+
+    public String getHeaders() {
+        /*
+         * x-rh-clientid: <id> x-rh-apitoken: <token> x-rh-insights-env: <env>
+         * 
+         * key1=val1;key2=v2;key3=v3
+         */
+        String clientId = System.getenv("BACKOFFICE_CLIENT_ID");
+        String token = System.getenv("BACKOFFICE_TOKEN");
```
Comment (bzwei) [inline src/main/java/com/redhat/management/approval/EmailDispatcher.java]: to play safe we need to trim off trailing `=` if exits.

File: src/main/java/com/redhat/management/approval/EmailDispatcher.java [hunk 3]
```diff
@@ -10,112 +10,114 @@
...
+        String clientEnv = System.getenv("BACKOFFICE_CLIENT_ENV");
+        
+        StringBuilder headers = new StringBuilder();
+        headers.append("x-rh-clientid=");
+        headers.append(clientId);
+        headers.append(";x-rh-apitoken=");
+        headers.append(token);
+        headers.append(";x-rh-insights-env=");
```
Comment (bzwei) [inline src/main/java/com/redhat/management/approval/EmailDispatcher.java]: only append `x-rh-insights-env` if it exists through ENV.

File: src/main/java/com/redhat/management/approval/Identity.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+/**
+ * This class was automatically generated by the data modeler tool.
+ */
+
+public class Identity implements java.io.Serializable {
+
+    static final long serialVersionUID = 1L;
+
+    private java.lang.String account_number;
```
Comment (bzwei) [inline src/main/java/com/redhat/management/approval/Identity.java]: can we rename to `accountNumber`?

File: src/main/java/com/redhat/management/approval/Internal.java [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+/**
+ * This class was automatically generated by the data modeler tool.
+ */
+
+public class Internal implements java.io.Serializable {
+
+    static final long serialVersionUID = 1L;
+
+    private java.lang.String org_id;
```
Comment (bzwei) [inline src/main/java/com/redhat/management/approval/Internal.java]: Same as above, can we change variable to camel case?

File: src/main/java/com/redhat/management/approval/Identity.java (hunk 4 above)
Comment (hsong-rh) [reply src/main/java/com/redhat/management/approval/Identity.java]: No, these field names will be used as key names for the header encoding later.

File: src/main/java/com/redhat/management/approval/Internal.java (hunk 5 above)
Comment (hsong-rh) [reply src/main/java/com/redhat/management/approval/Internal.java]: Same reason as above.

**Pull request merged**